### PR TITLE
`prefix` is missing for installing bash_completion

### PR DESCRIPTION
**Pull request by jean**
```
--- a/Makefile
+++ b/Makefile
@@ -37,7 +37,7 @@ install: git-hub git-hub.1 ftdetect.vim bash-completion README.rst
        install -m 644 -D git-hub.1 $(DESTDIR)$(prefix)/share/man/man1/git-hub.1
        install -m 644 -D ftdetect.vim \
                $(DESTDIR)$(prefix)/share/vim/addons/ftdetect/githubmsg.vim
-       -install -m 644 -D bash-completion $(DESTDIR)/etc/bash_completion.d/git-hub
+       -install -m 644 -D bash-completion $(DESTDIR)$(prefix)/etc/bash_completion.d/git-hub
        install -m 644 -D README.rst $(DESTDIR)$(prefix)/share/doc/git-hub/README.rst
```

### CODE REVIEW

**Comment by jean** [comment]
@leandro-lucarella-sociomantic 👀

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,9 @@
 
 prefix ?= /usr/local
+# fall back to $prefix/etc (like autoconf); see discussion on #222
+#sysconfdir ?= $(prefix)/etc)
```
Comment (llucax) [inline Makefile]: I don't think this provides anything useful, once you need to override `sysconfdir` a fallback atlernative doesn't really matter.

**Review by llucax — CHANGES_REQUESTED** [review]
Other than the tiny comment, LGTM.

**Comment by jean** [comment]
Updated :-)

**Comment by leandro-lucarella-sociomantic** [comment]
Thanks! I will take the liberty to squash both commits before merging :)

**Comment by leandro-lucarella-sociomantic** [comment]
Merged as b977c3c8d06904970dd59e023f418fac4a96cc5f.

**Pull request closed without merging**